packages/replay/src/canvas.ts

```diff
@@ -46,9 +59,32 @@ export class ReplayCanvas implements Integration {
       _experiments: {
         canvas: {
           ...this._canvasOptions,
-          manager: getCanvasManager,
+          manager: opts =>
+            this.canvasManager({
+              ...opts,
+              manualSnapshot: this._canvasOptions.manualSnapshot,
+            }),
         },
       },
     };
   }
+
+  /**
+   * Take manual snapshot of canvas
+   */
+  public snapshotCanvas(canvasElement?: HTMLCanvasElement): void {
```

Comment:
Shorter + consistent with canvas manager method

```suggestion
  public snapshot(canvasElement?: HTMLCanvasElement): void {
```